fli-iam/shanoir-ng
# shanoir-issue#2334: add console html component in keycloak page to display error

How to test:
On the "create account" page, use an already existing email address
Fill the rest of the form and click "Request Account"
An error should be displayed

![image](https://github.com/user-attachments/assets/63e8f100-7507-499e-a642-a35778e78f1c)
